dashboard/app/dsl/script_dsl.rb

```diff
@@ -303,6 +308,7 @@ def self.serialize_stages(script)
                 type,
```

Comment:
I think you will need to remove `type = 'named_level' if sl.named_level` above, otherwise when you serialize a script file it looks like you will get `named_level '...', named: true`.

even though there will be two acceptable representations of named levels in script files for a while, I think we will only want one output format when serializing a script. So, a reasonable outcome will be that if I edit a script with `named_level 'foo'` and then save it, that it would be converted to `level  'foo', named: true`.

this serialization logic should really be covered by a unit test. here is the only (!) example where we test script serialization which should provide a good starting point: https://github.com/code-dot-org/code-dot-org/blob/c5612b059c8162dbd0ac4cba5a5e04ff7bbbf439/dashboard/test/dsl/dsl_test.rb#L634

for a sanity test on your local machine, you should be able to exercise the script serialization path by making edits on your /s/.../edit page. this path also gets hit by `clone_with_suffix`, but if you have unit testing on serialization then that seems like adequate coverage for the script cloning case.